### ISSUE TITLE
The cityscapes training time is so long

### ISSUE THREAD

**Issue by YoungGer** [intel-isl/MultiObjectiveOptimization#4]
Thanks for your amazing code. I want to reproduce the cityscapes result.

I use one P40 card and run the training code. However, it doesn't finish one epoch in 4 hours.

I want to know if there are some problems and how long did you train the net in cityscapes dataset.

Thank you very much.

**Comment by ozansener** [comment]
There are two options using our efficient approximation vs not using. You need to set  `use_approximation=True` for fast training. 

As I mentioned in another issue, here is the full param set for Cityscapes training:

`optimizer=Adam|batch_size=8|lr=0.0005|dataset=cityscapes|normalization_type=none|algorithm=mgda|use_approximation=True`
